### DISCUSSION TITLE
cannot install  to laravel 8 application

### DISCUSSION THREAD

**Discussion by H-web268** [spatie/laravel-medialibrary#2445] [Q&A] [answered]
i try to install the spatie package to my laravel 8 application with php 8.0.3 but I got this error:
 [InvalidArgumentException]
  Package spatie/laravel-medialibrary at version ^8.0.0 has a PHP requirement incompatible with your PHP version, PHP
   extensions and Composer version

**Comment by MikeMazalahi** [comment, marked as answer]
I believe support for PHP8 was added in medialibrary 9.x.x, any particular reason why you're not using that?

**Comment by H-web268** [comment]
thank you very much Mike you are right now it works thank you
